src/server/hotkey_collector.h

```diff
@@ -69,6 +84,26 @@ class hotkey_collector
     void capture_hash_key(const dsn::blob &hash_key, int64_t weight);
     void handle_rpc(const dsn::replication::detect_hotkey_request &req,
                     /*out*/ dsn::replication::detect_hotkey_response &resp);
+
+private:
+    detect_result _result;
```

Comment:
I didn't find any usage about `detect_result _result` in this pr, maybe you can add it in next pr.